Aspose.Slides.WebExtensions.Tests/TestDemo.cs

```diff
@@ -0,0 +1,61 @@
+using Aspose.Slides;
+using Aspose.Slides.Export.Web;
+using Aspose.Slides.WebExtensions;
+using NUnit.Framework;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Xml;
+
+namespace Aspose.Slides.WebExtensions.Tests
+{
+    public class TestDemo
+    {
+        private string TemplatePath = null;
+        private string PresentationFilePath = null;
+        private string OutputPath = null;
+        private string RootDirectory = null;
+
+        [SetUp]
+        public void Setup()
+        {
+            RootDirectory = TestContext.CurrentContext.TestDirectory;
+            PresentationFilePath = Path.Combine(RootDirectory, "TestData", "demo.pptx");
+            TemplatePath = Path.Combine(RootDirectory, "templates", "single-page");
+            OutputPath = Path.Combine(RootDirectory, "single-page-demo-output");
+
+            using (Presentation pres = new Presentation(PresentationFilePath))
+            {
+                WebDocument document = pres.ToSinglePageWebDocument(TemplatePath, OutputPath);
+                document.Save();
+            }
+        }
+        static string FixVideoTag(string input)
+        {
+            input = Regex.Replace(input, "(<video[^>]+) controls>", "$1 controls=\"\">");
+            input = Regex.Replace(input, "<source[^>]+>", "$1 />");
+            return input;
+        }
+
+        [Test]
+        public void TestHeadlineConnector()
+        {
+            string indexHtmlPath = Path.Combine(OutputPath, "index.html");
+            Assert.IsTrue(File.Exists(indexHtmlPath));

```

Comment:
@polonair 

It seems that this assert is not related to the functional part of the test. It asserts something related to the test run itself (or, at least not relates to headline connector as test name assume). If it fails, it means that the problem somewhere in the test project, but not in the functionality under the test.